subworkflows/local/generate_downstream_samplesheets/tests/main.nf.test

```diff
@@ -0,0 +1,42 @@
+nextflow_workflow {
```

Comment:
This file can be removed, a) because nf-test isn't implemented in taxprofiler yet, and b) it wouldn't work anyway - you're giving TAR files to generate a MAG  samplesheet which doesn't make sense (I assume you copy-pasta'd and not yet updated)

Reply:
I added this to the PR by mistake 😬 WIll remove it.